frontend/src/utils/CampUtils.ts

```diff
@@ -56,19 +56,42 @@ export const getCampStatus = (camp: CampResponse): CampStatus => {
   return CampStatus.COMPLETED;
 };
 
-export const getFormattedDateString = (dates: Array<string>): string => {
-  const startDate = new Date(dates[0]).toLocaleDateString("en-us", {
+export const sortDateStrings = (unsortedDates: string[]): string[] => {
+  const dateNumToStrMap: Map<number, string> = new Map();
+  unsortedDates.forEach((curDate: string) => {
+    const parsedDate = Date.parse(curDate);
+    dateNumToStrMap.set(parsedDate, curDate);
+  });
+  const dateNumToStrArr = Array.from(dateNumToStrMap);
+  dateNumToStrArr.sort();
+
+  const sortedDates: string[] = [];
+  dateNumToStrArr.forEach((date) => {
+    sortedDates.push(date[1]);
+  });
+
+  return sortedDates;
+};
+
+export const getFormattedDateString = (
+  dates: string[],
+  requireSorted = false,
```

Comment:
Added a requireSorted parameter with default value false. I don't think we ever need formatted date strings without sorting them, but if we did want the option, I didn't want to change/break anything else we had, thus the default value of false (and set it to true to get sorted dates).